### PR TITLE
chore(C): Add bubble sorting for C language

### PR DESCRIPTION
**Pull request by nasdev-cmd**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

Bubble sort for C language: https://onlinegdb.com/SJeYlj94wO

**Other information:**

### CODE REVIEW

**Review by Arsenic-ATG — DISMISSED** [review]

File: algorithms/C/sorting/bubble-sort.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
+//bubble sort
+#include <stdio.h>
+#include <stdlib.h>
+
+void swap(int *value1, int *value2);
+
+
+//time complexity: O(n^2)
```
Comment (UG-SEP) [inline algorithms/C/sorting/bubble-sort.c:8]:
```suggestion
It is better if you add time complexity at the beginning or end of the program
```

File: algorithms/C/sorting/bubble-sort.c [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    bubble_sort(array, array_size);
+    
+    printf("Sorted array: [ ");
+    for(int i = 0; i < array_size; i++)
+    {
+        printf("%d ", array[i]);
+    }
+    printf("]\n");
+    return 0;
+}
```
Comment (UG-SEP) [inline algorithms/C/sorting/bubble-sort.c:54]: please give a example at the end of the program i.e. input 
                                                                                    output

**Review by UG-SEP — CHANGES_REQUESTED** [review]
Kindly make the changes
Feel free to ask doubt

**Review by UG-SEP — APPROVED** [review]
@nasdev-cmd Thanks for contributing 😀
We glad to have a contributor like you 😊
Hope you contribute more to this project 🤞🤞

**Comment by UG-SEP** [comment]
@ming-tsai please check it

**Comment by nasdev-cmd** [comment]
@UG-SEP Thanks, I look forward to collaborating with you!

**Comment by UG-SEP** [comment]
> @UG-SEP Thanks, I look forward to collaborating with you!

Sure

**Review by Arsenic-ATG — APPROVED** [review]

**Pull request merged**